oak-blob-cloud-azure/src/test/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzureCachingDataStoreTest.java

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
The super class adds methods for local testing. If the kind of tests that are covered there are covered in other areas it might be good to have it remain as a local test which will add some coverage for the AzureDatastore class.